Venmo/src/test/java/com/braintreepayments/api/venmo/VenmoApiUnitTest.java

```diff
@@ -67,9 +67,6 @@ public void createPaymentContext_createsPaymentContextViaGraphQL() throws JSONEx
         verify(braintreeClient).sendGraphQLPOST(captor.capture(), any(HttpResponseCallback.class));
 
         JSONObject graphQLJSON = captor.getValue();
-        String expectedQuery =
```

Comment:
These fail with the Kotlin multiline string indentation. I think since they are set string values in the code it doesn't really add value to include them in the unit tests. The alternative would be to convert the whole test file to Kotlin to change these to use the multiline spacing - what do you think?

Reply:
Ah I see. I think we can leave it as a follow up todo.